extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.example.3.json

```diff
@@ -0,0 +1,12 @@
+{
+  "xdm:messageID": "",
+  "xdm:channel": {
+    "xdm:_id": "Mobile Push",
+    "xdm:_type": "https://ns.adobe.com/xdm/channel-types/push"
+  },
+  "xdm:campaign": {
+    "xdm:campaignID": "19d3b8d6-32fa-4b06-b5fe-2692f828072b",
+    "xdm:campaignName": "My Push Campaign",
+    "xdm:campaignVersionID": "7b203379-1d61-42ef-a096-aded7f957d62"
+  }
+}
```

Comment:
Please change the namespaces in your example files as shown below:

{
  "https://ns.adobe.com/experience/journeyOrchestration/classification/messageID": "",
  "(https://ns.adobe.com/experience/journeyOrchestration/classification/channel": {
    "xdm:_id": "Mobile Push",
    "xdm:_type": "https://ns.adobe.com/xdm/channel-types/push"
  },
 "https://ns.adobe.com/experience/journeyOrchestration/classification/campaign": {
    "xdm:campaignID": "19d3b8d6-32fa-4b06-b5fe-2692f828072b",
    "https://ns.adobe.com/experience/journeyOrchestration/classification/campaignName": "My Push Campaign",
    "https://ns.adobe.com/experience/journeyOrchestration/classification/campaignVersionID": "7b203379-1d61-42ef-a096-aded7f957d62"
  }
}`

Reply:
Thank you. Fixed.